app/controllers/index.js

```diff
@@ -14,17 +14,14 @@ export default class IndexController extends Controller {
 
   @computed('filteredEvents.[]')
   get callForSpeakersEvents() {
-    return this.filteredEvents.filter(event => {
-      const callForPapers = event.get('speakersCall');
-      const sessionEnabled = event.isSessionsSpeakersEnabled;
-      if (!callForPapers || !callForPapers.get('startsAt')  || !callForPapers.get('endsAt')) {
-        return false;
-      }
-      const startDateTime = callForPapers.get('startsAt');
-      const endDateTime = callForPapers.get('endsAt');
-      const privacyState = callForPapers.get('privacy');
-      return (moment().isBetween(startDateTime, endDateTime) && (sessionEnabled) && (privacyState === 'public'));
+    let filteredEventsCfs = this.filteredEvents.filter(event => {
```

Comment:
You can directly pass `isEventCfsOpen` in filter function, no need to create an arrow function

Reply:
Done